sidebars.js

```diff
@@ -1,52 +1,18 @@
 module.exports = {
   docs: [
     'introduction',
-    'quick-start',
+    `security/disclosure`,
```

Comment:
Nit: I would prefer for the disclosure to be at the bottom, it is important, but giving it too much prominence may send the message that we don't trust the project state, which isn't true. (Also, I would prefer it to be named "Security disclosure").

Reply:
Applied suggestion